src/Cake.Npm/NpmVersionAliases.cs

```diff
@@ -0,0 +1,71 @@
+namespace Cake.Npm
+{
+    using System;
+    using Cake.Npm.Version;
+    using Core;
+    using Core.Annotations;
+
+    /// <summary>
+    /// Npm Version aliases
+    /// </summary>
+    [CakeAliasCategory("Npm")]
+    [CakeNamespaceImport("Cake.Npm.Version")]
+    public static class NpmVersionAliases
+    {
+        /// <summary>
+        /// Versions all packages for the project in the current working directory.
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <example>
+        /// <code>
+        /// <![CDATA[
+        ///     NpmVersion();
+        /// ]]>
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Version")]
+        public static string NpmVersion(this ICakeContext context)
+        {
+            if (context == null)
+            {
+                throw new ArgumentNullException(nameof(context));
+            }
+            
+            return context.NpmVersion(new NpmVersionSettings());
+        }
+
+        /// <summary>
+        /// Versions packages using the speVersionfied settings.
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <param name="settings">The settings.</param>
+        /// <example>
+        /// <para>Use speVersionfic log level ('npm Version')</para>
+        /// <code>
+        /// <![CDATA[
+        ///     var settings = new NpmVersionSettings();
```

Comment:
Should set a property on the settings instance

Reply:
removed the alias that uses the settings has there aren't any settings properties to set for `npm version`